keyboards/dz60/keymaps/heather/keymap.c

```diff
@@ -0,0 +1,76 @@
+#include QMK_KEYBOARD_H
+
+//Layers
+#define _DEFAULT 0
+#define _FN 1
+#define _MEDIA 2
+
+//Filler for KC_TRNS/KC_NO just so the key maps are a bit more readable
+#define ______ KC_TRNS
+#define XXXXXX KC_NO
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* Qwerty Base layer
+ * ,-----------------------------------------------------------------------------------------.
+ * | Esc |  1  |  2  |  3  |  4  |  5  |  6  |  7  |  8  |  9  |  0  |  -  |  =  |Bspc | Del |
+ * |-----------------------------------------------------------------------------------------+
+ * | Tab    |  Q  |  W  |  E  |  R  |  T  |  Y  |  U  |  I  |  O  |  P  |  [  |  ]  |   \    |
+ * |-----------------------------------------------------------------------------------------+
+ * | `/_Media|  A  |  S  |  D  |  F  |  G  |  H  |  J  |  K  |  L  |  ;  |  '  |    Enter    |
+ * |-----------------------------------------------------------------------------------------+
+ * |   Shift   |  Z  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  |  /  |Shift| Up  |Prsc |
+ * |-----------------------------------------------------------------------------------------+
+ * | LCtrl | LGui  | LAlt  |     Space    |  Fn  |    Space    |Home | End |Left |Down |Right|
+ *  `----------------------------------------------------------------------------------------'
+ */
+	[_DEFAULT] = LAYOUT_2_shifts(
```

Comment:
It looks like the keymap is having issues with `LAYOUT_2_shifts`.

Reply:
`LAYOUT_2_shifts` was removed in 2018; use `LAYOUT_all` instead.